Sources/CodeGenerator/Stages/GenerationStage/TemplateFiller/TemplateFiller.swift

```diff
@@ -10,5 +10,9 @@ import Foundation
 public protocol TemplateFiller {
     /// Load template at given path and fill it with provided context
     /// Returns filled template  text
-    func fillTemplate(at path: String, with context: [String: Any]) throws -> String
+    func fillTemplate(
+        at path: String,
+        with context: [String: Any],
+        specificationRootPath: String
```

Comment:
Are you shire that it is good decision to pass this value into method? I think that it would be much better to store it insode scpecific implementation of filter. Because I guess that there are limited number of filters which are interested in this variable

Reply:
Thank you, fixed